### ISSUE TITLE
DBT-core 1.3 incompatibility

### ISSUE THREAD

**Issue by imkehno** [mjirv/dbt-datamocktool#55]
### Describe the bug
`dbt_datamocktool.unit_test` does not compile after upgrading to dbt-core 1.3. `dbt test` results to SQL compilation error. 

### Steps to reproduce
Upgrade dbt-core -> 1.3 and `dbt test`

### Expected results
Would expect `dbt test` to run with success.

### Actual results
SQL Compilation error.

### System information
**The contents of your `packages.yml` file:**

**Which database are you using dbt with?**
Snowflake

**The output of `dbt --version`:**
```
Core:
- installed: 1.3.0
- latest:      1.3.0 - Up to date!

Plugins:
- snowflake: 1.3.0 - Up to date!
```

### Additional context
I'm quite certain that this is due to a new feature addition in dbt-core: `python_model`, which allows `dbt run -m` to execute files with .py extension.

Feature and the breaking changes were initially added here. Name of the `Node` attribute `raw_sql` is changed to `raw_code`:
https://github.com/dbt-labs/dbt-core/pull/5421

And this leads to issues as `get_unit_test_sql()` macro uses the original attribute to fetch compiled sql. Quick fix to verify: `{% set ns.test_sql = ns.graph_model.raw_code %}`
https://github.com/mjirv/dbt-datamocktool/blob/4ad7486884c6242152078551f376c42dcaf90562/macros/dmt_get_test_sql.sql#L24

### Are you interested in contributing the fix?
Sure. Though quick fix breaks backwards compatibility.

**Comment by mjirv** [comment]
Thanks for raising this and looking in to what's going on!

I'll spend some time to see if there's a backwards-compatible fix, but if not, I think it's ok to make breaking changes. People on earlier versions of dbt Core will just have to update the package version they use.

**Comment by imkehno** [comment]
Our fork seems to be working nicely (though it's just a fix for >=1.3, as mentioned). I can make a PR, unless you want to investigate more @mjirv

**Comment by mjirv** [comment]
Resolved by #57
